modules/http4s/src/main/scala/com.snowplowanalytics.iglu/client/resolver/registries/Http4sRegistryLookup.scala

```diff
@@ -134,10 +134,8 @@ object Http4sRegistryLookup {
           RegistryError.RepoFailure(error).asLeft
         }
       case Status.ClientError(response) =>
-        response.bodyText.compile.string.map { body =>
-          val error = s"Unexpected server response: $body"
-          RegistryError.ClientFailure(error).asLeft
-        }
+        val error = s"Unexpected server response: ${response.status.code}"
```

Comment:
Maybe "Unexpected response code" is a better message?

Reply:
definitely, not every response comes from server